src/Sulu/Bundle/TestBundle/Behat/DefaultContext.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TestBundle\Behat;
+
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+
+/**
+ * Default context class for Sulu contexts.
+ */
+class DefaultContext extends BaseContext implements SnippetAcceptingContext
```

Comment:
Ah, ok, I thought you introduced another Interface in Sulu, then I would have been missing the `Interface`-Suffix ;-)